### DISCUSSION TITLE
Soundtrack

### DISCUSSION THREAD

**Discussion by shil5** [github/india#8] [Satellite India 2021 Q&A]
Not an important question.
But I really would like to know what are the soundtracks you used in the beginning of the session, during the wait time BEFORE 9.30 am? :) Thank you and apologies for filling up the discussion forum.

Edit: I do know that the music played after 9.30 was ORCA action but I actually want to know about the music they played BEFORE 9.30 AM during the wait time.

**Comment by khoparzi** [comment]
I'm loving the chilled out ORCA action by Beatnyk right now.

**Reply by msupernaut** [reply]
If you have headphones, it is just amazing. the beats are bouncing off from ear-to-ear!

**Reply by LokeshC2** [reply]
I wasn't familiar with ORCA before this, so watching it live really is something.

**Reply by sahilcool-nsut** [reply]
How exactly is it working, can some1 explain in brief :O? it looks and sounds awesome

**Reply by tsadarsh** [reply]
Yes, I too am clueless about ORCA.

**Reply by shil5** [reply]
Me too and fantastically enough, I really liked the music that they played before 9.30AM as well. Did you recognize any of them?

**Reply by sahilcool-nsut** [reply]
Just looked up a little, ORCA seems beautifully innovative 💯

**Reply by khoparzi** [reply]
The way ORCA works for most people is that you have a sampler or midi instrument and use the 'code' in ORCA to sequence parts and modulations.

Essentially it's a form of live coding practice.

**Reply by sahilcool-nsut** [reply]
So they wrote the code and its like compiling or running live time rn, they aren't doing anything live or are they?

**Reply by khoparzi** [reply]
It's like running a REPL in Python or Clojure.

EDIT: Or more like a hybrid between a mod tracker you would use for music in game dev and a live coding platform.

**Reply by karthiksedoc** [reply]
> Me too and fantastically enough, I really liked the music that they played before 9.30AM as well. Did you recognize any of them?

It's internal, you can not find it outside  ;)

**Reply by beatnyk** [reply]
> How exactly is it working, can some1 explain in brief :O? it looks and sounds awesome

Hey. ORCA is not standalone and needs a MIDI-enabled sound source to trigger sounds. The alphabets and numbers are MIDI information triggering the source (Ableton in this case) link internally using LoopMIDI. You can learn more about it on https://100r.co/site/orca.html

**Comment by GowthamGoush** [comment]
It's really soothing to hear ☺️

**Reply by khoparzi** [reply]
@beatnyk really nails the chill factor

**Comment by tsadarsh** [comment]
Is this ORCA performance related to http://www.studioorca.com/ in anyway?

**Reply by Shathin** [reply]
It is most likely this one - https://github.com/hundredrabbits/Orca

**Reply by khoparzi** [reply]
You got it! Its a fun environment to make music in. They have some other amazing creative coding projects as well.

**Reply by kshitij12345** [reply]
Great. Thansk for the link!

**Comment by Nomita92** [comment]
this music is so relaxing

**Reply by shil5** [reply]
Yes, and the bass is like damn.
